### ISSUE TITLE
Support for Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by coreydeli** [atosatto/ansible-dockerswarm#81]
Looking for support for Ubuntu 20.04 LTS as we have moved to this platform for docker swarm. Current project does not support it, can downgrade if needed though we would rather utilize this version.

**Comment by fd98279** [comment]
I installed docker swarm cluster on 20.04. Lastest galaxy version works partially:
skip_docker_py: true <- Had to disable this since python-pip package is not found in ubuntu 20.04
skip_docker_compose: true <- Did not install this, since it's no needed on the server.

```yaml
  - name: Install docker
    include_role:
      name: atosatto.docker-swarm
      apply:
        tags:
          - docker
    vars:
      ansible_become: yes
      skip_docker_compose: true
      skip_docker_py: true
```

**Comment by juanluisbaptiste** [comment]
There is a PR fixing the [pip package install](https://github.com/atosatto/ansible-dockerswarm/pull/92/files) on ubuntu >= 20.04, but don't get your hopes up that it will be marged as this project seems to be abandoned.

I have a [fork if this repo](https://github.com/juanluisbaptiste/ansible-dockerswarm) which fixes this, adds support for ubuntu 20.04 and also fixes issues #78 and #96 if you want to give it a try.
